### ISSUE TITLE
🤔 [QUESTION]

### ISSUE THREAD

**Issue by revidovich** [antvis/G2Plot#3055]
### 🐛 Question description
![image](https://user-images.githubusercontent.com/63742797/147565281-ad8f0288-c465-43c9-b047-e44ad6a0eb47.png)  
  
Good day! I'm sorry if this question has already been asked, I didn't find similar examples in the documentation. so..
Does your library have the ability to create such graphics? - there are many graphs in one... and has many Y-axis lines for each graph.

**Comment by ntscshen** [comment]
Non-universal chart，customization is inevitable for business products。
https://g2plot.antv.vision/en/docs/manual/plugin

**Comment by revidovich** [comment]
do you have any examples ? maybe)

**Comment by ntscshen** [comment]
I customized it based on G2 (flexible enough). In advanced features (there are enough examples),
document internationalization may not be friendly enough. If you are interested to try
https://antv-g2.gitee.io/en/docs/api/advanced/register-shape

---------
If you need more detailed information, you need the help of the real developers here

**Comment by revidovich** [comment]
we customized shapes and colors, but the chart only have 1 x-axe and 1 y, seems like we cant customize linear chart like that..   
and its absolutelly true that document internationalization is rather unfriendly))   
![image](https://user-images.githubusercontent.com/63742797/147644745-c7b69904-90c2-4174-ac61-e0a0a34e52ee.png)
